Module4_Labs/Lab5_nColorable/Cards/11.md

```diff
@@ -4,11 +4,11 @@
 
 <!--badges={Python:15,Algorithms:15}-->
 
-This lab will require you to use **Depth First Search** and **Backtracking Search** to solve the n-Colorable problem. Therefore, I will give you some explanation on both of these concepts.
+This lab will require you to use **Depth First Search** and **Backtracking Search** to solve the n-Colorable problem. Therefore, we will briefly review both of these concepts.
```

Comment:
this lab should have an introductory card on how backtracking search works, appropriately placed ofc, and none of that context should be in an medium card